evap/evaluation/tools.py

```diff
@@ -93,7 +93,7 @@ def send_publish_notifications(courses, template=None):
 
     for course in courses:
         # for courses with published grades, all contributors and participants get a notification
-        if course.has_enough_voters_to_publish_grades:
+        if course.has_enough_voters_to_publish_grade:
```

Comment:
i'd say an email is only necessary if we have reached the significance threshold. otherwise there is something to see, if the users browse through the data, but a notification feels strange when at the same time we say "the results are not significant".

Reply:
okay. please change the comment, at least "grade" not "grades",